README.md

```diff
@@ -193,28 +210,6 @@ If you are not interested in the standardized mitogenome sequence(s) or MITOS an
 
 
 
-### Summarize mtGrasp results
```

Comment:
I thought that the script just needed some fixes? Why are we deleting it altogether?

Reply:
I will add this section once i have the summary.py is ready

Reply:
Added summarize.py.